pywatts/modules/feature_extraction/statistics_extraction.py

```diff
@@ -0,0 +1,93 @@
+from enum import IntEnum
+from typing import Dict, Optional, List
+
+import xarray as xr
+import numpy as np
+from pywatts.core.base import BaseTransformer
+from pywatts.utils._xarray_time_series_utils import _get_time_indexes
+
+
+class StatisticFeature(IntEnum):
+    """
+    The available statistic features, that are extractable by the statistic extraction module:
+
+    min: Extracting the min of the time series sample.
+    max: Extracting the max of the time series sample.
+    std: Extracting the std of the time series sample.
+    mean: Extracting the mean of the time series sample.
+    """
+    min = 1
+    max = 2
+    std = 3
+    mean = 4
+
+
+class StatisticExtraction(BaseTransformer):
+    """
+    This module extract statistical features based on samples of timeseries defined by a DataArray input.
```

Comment:
This module extracts statistical features based on samples of time series defined by a DataArray input.